src/components/faceVerification/screens/IntroScreen.js

```diff
@@ -112,6 +114,23 @@ const IntroScreen = ({ styles, screenProps, navigation }) => {
     fullName,
   ])
 
+  useEffect(() => {
```

Comment:
why just on this screen ? make it a separate hook and add on all screen used in FV standalone

Reply:
also this isn't what exactly I meant. look at src/Router.js (example how we use navigation state change for send analytics event `GOTO`)

Reply:
1. only on this screen because all other 3 screens lead back here (goToRoot)
2. when you say navigation events, this is what came back from the docs
3. ok, I will recheck again

Reply:
how about onWillBlur ?